code/modules/cargo/packs.dm

```diff
@@ -2439,6 +2439,31 @@
 					/obj/item/clothing/head/helmet/bluetaghelm)
 	crate_name = "laser tag crate"
 
+/datum/supply_pack/costumes_toys/knucklebone
+	name = "Knucklebone Game Crate"
+	desc = "A fun dice game to keep you entertaned for hours and is totaly not ripped off from another game!"
```

Comment:
*entertained

I also feel the need to point out knucklebones is a real ancient game and cult of the lamb did not invent it so nobody is going to get this reference in a few months

Reply:
I am actually extremely glad to hear that!

Reply:
Recommend it be "A fun dice game definitely not invented by a cult. Consult your local chaplain regarding approved religious activity."